Fix configure error on macOS which does not have /bin/true.
https://sourceforge.net/p/kcd/patches/2/
--- version.cc.orig	2009-02-18 06:01:06.000000000 -0600
+++ version.cc	2019-11-04 07:41:19.000000000 -0600
@@ -1,4 +1,4 @@
-//bin/true; VERSION="7.15.0"; return 0
+//usr/bin/env true; VERSION="7.15.0"; return 0
 // Above magic is for configure to get version number
 // Generate automatically from scripts/versions.cc.src
 char	version[] = "7.15.0";
